### ISSUE TITLE
Viscerators should be hostile to everyone

### ISSUE THREAD

**Issue by MasterofNothing** [d3athrow/vgstation13#18204]
Viscerator grenades are terribly unbalanced. They're pretty robust, and for that very reason, they were initially supposed to attack everyone - including those who used it. Over the years, people have wined over getting wrecked by them, and some coders changed the code to make them friendly to the antag party.

The result is that if you can somehow get a lot of viscerator grenades (e.g. as a mechanic), you will win the round, unless you make a dumb mistake. Just spam them in the hallway and you're set.

It's a lot better if you must use them tactically. That makes it impossible to spam them in every hallway and department and still achieve your antag objectives.

In other words, make them hostile to everyone, please.

Related issues: 
[#9808](https://github.com/vgstation-coders/vgstation13/issues/9808)
[#12404](https://github.com/vgstation-coders/vgstation13/pull/12404)
And probably more...

**Comment by Irradiation** [comment]
Viscerators are "generally" only robust when in high numbers, which is exactly what happened last round. As for having access to a metric fuckton, that's kind of the Mechanic's gimmick. His syndie device analyzer is meant to allow you to mass produce all kinds of broken things. As for them being strong, the last time I've ever heard of them being powerful was when somebody tried the butterfly knife as a wizard and "found out" that you could make an almost infinite amount.

That said, the current cost of viscerators already makes their use tactical, it's just that Mechanics can sidestep that issue, as it's their gimmick. 

"You will win the round unless you make a dumb mistake". The same can be said for dude bombs lmao, plasmaflooding, subverting the silicons, etc.

**Comment by MasterofNothing** [comment]
At the very least, I think it's a good idea to get a few opinions in. I think it's too unbalanced, and that even in the absence of the possibility to mass-produce them, they are still unbalanced. They should attack everyone, or be easier to kill. I.e. they should act exactly as they did before.

**Comment by DrSnips** [comment]
I have never seen people complain about the current state of them until here and I personally think they are actually pretty baf since they die to one swing of a toolbox. I even made a buffed version of them, to basically a reskined spacecarp, and they still feel a bit underwhelming save from the one time when a bug was present when it was first added.

**Comment by MasterofNothing** [comment]
They do not die to one swing of a toolbox.

**Comment by DrSnips** [comment]
https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/code/modules/mob/living/simple_animal/hostile/viscerator.dm
Except you're completely wrong. Toolboxes do 15 and visc. have 15 health.
Unless you're looking at the special reskined spacecarp ones for the butterfly knife.

**Comment by Aurx** [comment]
Angry dead guy whines about "OP" thing he thinks is stronger than it is, news at 11.

**Comment by MasterofNothing** [comment]
Yeah, yeah, it's the "attack whoever's presenting a gameplay/balance issue" shit again. If you'd get your head out of your ass, it's pretty easy to see that I wasn't the only one not happy about the viscerators, nor the only one expecting that they would attack everyone - including the guy who spawned them. Even he thought they would. Why did they think that? Because it used to be like that. And in my view, it created a nice balance. 

It's a given that people whine about dying, but how many balance issue tickets have you seen me open in the past years? I think it's a legitimate problem, hence the post. It's an incredibly easy and overpowered tactic that's just begging to be abused further. When that happens, feel free to dig up this thread.

Mechanic + eviscerators + resources = win as long as the mechanic has enough of these grenades before getting outed. That this isn't a common issue is simply due to most traitor mechanics not knowing their heads from their asses... But do you really want to make it so easy for those who do?

And regarding the toolbox kills - they're pretty easy to miss, which is why I thought they needed two hits (and I didn't check code to see their health). When there's two/three/four/five, you're fucked without proper protection anyway.

If everyone else here agrees that it's not an issue, fine.

**Comment by DrSnips** [comment]
Are you feeling antagonized? The hacked scanner is seen as a "go fucking wild" ticket since it does have the ability to replicate almost anything. The whole point as an antag and the scanner is to fuck with the station.

**Comment by MasterofNothing** [comment]
It's basically as difficult to pull off as a holo fire used to be. I'm a bit surprised that you think that's a good game mechanic to keep.

Antags should fuck with the station, yes. But it should be challenging, and not an easy win.

**Comment by Reviire** [comment]
I think they should have an increasing chance to lose their antag friendliness the more they attack people and acquire the taste for blood. But beyond that, no. 

We already have far, far easier things in the game that should be balanced first. Heavily nerfing things that are miles worse than the big killers is just limiting peoples options, and making blasma/subvert/bombs lmao a better option.

EDIT: Also get a combat shotgun with buckshot

**Comment by iFlashYou** [comment]
It killed me, nerf it!

**Comment by Blithering** [comment]
As a fellow syndicate who had a pretty fun gimmick cut short by the viscerators that round, they're fine. Unless it becomes an "every time I get antag" situation like when people discovered how easy it was to crank out 6 Max caps and delete the station ten minutes in I don't think anything needs to change.

**Comment by MasterofNothing** [comment]
OK, thanks for the input everyone. Overal consensus is that it's not a problem. I'll mark this closed. If it's something that'll be a problem in the future, I'm sure there will be more discussion then.

I also did not see [#5715](https://github.com/vgstation-coders/vgstation13/pull/5715) until just now.

**Comment by jknpj** [comment]
🤔

**Comment by MasterofNothing** [comment]
I figured that, as a result of a recent round, there could be one or two people who want to give their opinion. If not, I'll just close it again.

**Comment by Reviire** [comment]
Close this again because they're still fine

**Comment by MasterofNothing** [comment]
I'm sure you can stomach waiting a little bit. There's the possibility that someone who actually still plays the game may weigh in.

**Comment by Irradiation** [comment]
Since this issue is buried somewhere deep on github, I suggest making a motherfucking PR to instantly get the attention of people so you can get extra voices on the topic so to speak. Or you can start a new issue and link this one and see what happens.

**Comment by MasterofNothing** [comment]
I posted it in OOC after the round, and in the thread. I don't want to make a big deal out of it, but if someone wants to weigh in, he/she/it can do it. If no one does, I'll just close it again.

**Comment by Exxion** [comment]
Honestly, this proposed change isn't really a solution to the issue at hand. It just means that the traitor would *also* die, not that everyone else wouldn't. It would pretty much just accomplish making viscerators more obnoxious to use.
If it really is such a problem (and I wasn't there, so I can't speak on whether or not it was), a better solution would be to make the viscerator grenade not scannable even by traitor mechanics

**Comment by MasterofNothing** [comment]
Well, no one in the round was annoyed enough to comment here, so closed again. Honk.
